src/index.js

```diff
@@ -3,26 +3,41 @@ import './assets/kebab-menu.png';
 import './assets/trash-can-icon.png';
 import './assets/refresh_reload_icon.png';
 
-const tasks = [
-  { description: 'Task 1', completed: false, index: 1 },
-  { description: 'Task 2', completed: true, index: 2 },
-  { description: 'Task 3', completed: false, index: 3 },
-];
+let tasks = [];
+
+function saveTasksToLocalStorage() {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+function updateTaskIndexes() {
+  tasks.forEach((task, index) => {
+    task.index = index + 1;
+  });
+}
 
 function renderTasks() {
+  // eslint-disable-next-line no-use-before-define
+  const deleteTask = (index) => {
+    tasks.splice(index, 1);
+    updateTaskIndexes();
+    renderTasks();
+    saveTasksToLocalStorage();
+  };
+
   const taskList = document.getElementById('task-list');
   taskList.innerHTML = '';
 
   tasks.forEach((task, index) => {
     const listItem = document.createElement('li');
-    listItem.draggable = true; // Enable draggable for list item
+    listItem.draggable = true;
 
     const checkbox = document.createElement('input');
     checkbox.type = 'checkbox';
     checkbox.checked = task.completed;
     checkbox.addEventListener('change', () => {
       tasks[index].completed = checkbox.checked;
       renderTasks();
+      saveTasksToLocalStorage();
     });
 
     const label = document.createElement('label');
```

Comment:
- [ ] If you see the project requirement you should create the function for editing the task and make sure to save the data in local storage.
- [ ] For every functionality I would suggest you please make a new file for new functionality So it would be easier to debug and the code looks clean as well.
### Requirement
![Screenshot 2023-06-14 175442](https://github.com/melybee-07/To-Do-List/assets/81727988/afc55a3a-de48-4962-aed7-7d7bf24ed297)